### DISCUSSION TITLE
Choam's support is currently suspended.

### DISCUSSION THREAD

**Discussion by cowboycodr** [cowboycodr/choam#40] [Announcements]
# Choam's support is currently suspended. 

## Reasoning
Quite frankly, I was tired of working on Choam. The number of people who found it useful was slim, and the downloads kept going down. 

For the people who are already using Choam, the final update will have `--destroy` which will remove Choam's existence from the project completely. However, will keep the structure. 

## Thank you!
To the people that did download Choam, I thank you! You guys were my motivation to continue this for a month. Choam has reached 13k downloads and I appreciate each individual one of them. 

However, maintaining choam constantly is tiring and repetitive. 

Thanks everyone.

**Comment by cowboycodr** [comment]
Do note: If I ever feel like coming back to this project I will, and I will announce it to everyone.

**Comment by svalordev** [comment]
:(

**Reply by cowboycodr** [reply]
Don't worry. I will return in a week-or-so. 

Currently, with school, it's become too much. And I just need a break to recoup and implement some new features. 

It will be back, and better than ever!
